Thesis.tex

```diff
@@ -178,9 +177,13 @@ \subsection{Open Source Software}
 
 The programmers themselves are often paid to work on the OSS, as the same report anouncement also states: "The number of paid developers is on the rise, as companies aggressively recruit top Linux talent. More than 80 percent of kernel development is done by developers who are being paid for their work. Volunteer developers tend not to stay that way for long."
 
+\subsection{Software Licensing}
+\label{ssec:SLic}
+Software is often made available using licenses. According to Merriam-Webster, the general meaning of a license is a "permission to act", respectively a "freedom of action"\parencite{WebstLic}. One of the subordinate meanings, again according to Merriam-Webster is "a grant by the holder of a copyright or patent to another of any of the rights embodied in the copyright or patent short of an assignment of all rights". 
 
-\subsection{Closed Source Software}
-\label{ssec:CSS}
+In the software business, the meaning is that you do not buy a copy of the software, but the rights to use the copy of the software, or as defined in Wikipedia: "A typical software license grants the licensee, typically an end-user, permission to use one or more copies of software in ways where such a use would otherwise potentially constitute copyright infringement of the software owner's exclusive rights under copyright law."\parencite{WikiSoftLic}. To be entitled to use such proprietary software, the users typically have to agree to compensate the copyright holder for obtaining a copy, which seems quite reasonable. 
+
+Furthermore, however, the user is most often required to agree to software license agreements, stating further limitations to which the license of the software is subject to. This mechano is summarized in Wikipedia as following: "Proprietary software licenses often proclaim to give software publishers more control over the way their software is used by keeping ownership of each copy of software with the software publisher. By doing so, Section 117 does not apply to the end-user and the software publisher may then compel the end-user to accept all of the terms of the license agreement, many of which may be more restrictive than copyright law alone."\parencite{WikiSoftLic} These terms might e.g. limit the duration of the license to one year, prolonged only in case of further payment. Microsoft e.g. grants the rights to use Office 365 only for the duration of a year, after which a repeated payment is required\parencite{Off365}.
```

Comment:
Either use "Furthermore" or "However" - Both together are unnecessary and cumbersome